doc/releases/release-notes-3.7.rst

```diff
@@ -834,15 +834,15 @@ Drivers and Sensors
   * Added support for Ambiq Apollo3 series.
   * Added support for multiple instances of the SPI NOR driver (spi_nor.c).
   * Added preliminary support for non-erase devices with introduction of
-    device capabilities to c:struct:`flash_parameters` and the utility function
```

Comment:
Kind of weird to be modifying release notes of an already finalized release.

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/79994#issuecomment-2423233033 for some context